packages/client/hmi-client/src/components/navbar/tera-navbar.vue

```diff
@@ -461,22 +462,33 @@ function closeLogoutDialog() {
 	isLogoutDialog.value = false;
 }
 
+function getNavMenuItem(project) {
+	return {
+		label: project.name,
+		icon: 'pi pi-folder',
+		command: () =>
+			router.push({
+				name: RouteName.Project,
+				params: { projectId: project.id, pageType: ProjectPages.OVERVIEW }
+			})
+	};
+}
+
 watch(
 	() => useProjects().allProjects.value,
 	() => {
 		const items: MenuItem[] = [];
+		const lastProjectUpdated = orderBy(useProjects().allProjects.value, ['updatedOn'], ['desc'])[0];
+
 		useProjects().allProjects.value?.forEach((project) => {
-			items.push({
-				label: project.name,
-				icon: 'pi pi-folder',
-				command: () =>
-					router.push({
-						name: RouteName.Project,
-						params: { projectId: project.id, pageType: ProjectPages.OVERVIEW }
-					})
-			});
+			if (project?.name !== lastProjectUpdated.name) {
```

Comment:
Maybe removing the recent project out of the project array like you did in the Add to project popup would be more efficient than having this if statement